docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+services:
+  bitcoind-miner:
+    container_name: bitcoin-miner
+    # image: alpen_signet:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    # env_file: 
+    #   .env
+    environment:
+      RPCUSER: user
+      RPCPASSWORD: password
+      MNEMONIC: "sense xxx xxx bone"
+      MINERENABLED: "1"
+      MAXTXFEE: "10"
+      SIGNETCHALLENGE: 
+      CONF_CONTENT: |
```

Comment:
Can we not pass the file content as env variables? We can do volume mounts for this.